src/redis-check-rdb.c

```diff
@@ -321,6 +322,8 @@ int redis_check_rdb(char *rdbfilename, FILE *fp) {
     } else {
         rdbCheckError("Unexpected EOF reading RDB file");
     }
+err:
```

Comment:
Hi ping @oranagra 
I am wondering the eoferr. Is it missing a return value and fclose(fp) ? And i check the code, the comment `with a fatal exit` seems to be wrong.

if so, i can chang it

Reply:
@enjoy-binbin
since the `err` label code doesn't have a return or another `goto`, it falls through to the code of the `err` label, and executes the `fclose` and return.
or am i missing your point?

Reply:
.... i see. After `eoferr` label, it will execute `err` label.  i did not know before... 
Both C or goto is rarely used in my work.
sorry for wasting your time. And really thanks for your reply.